sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md

```diff
@@ -10,6 +10,14 @@
 
 ### Other Changes
 
+## 2.8.1 (2023-12-04)
+
+### Other Changes
+
+#### Dependency Updates
+
+Regular updates for dependency versions.
```

Comment:
Is there a particular reason we are not specific with this library's dependencies?

Reply:
The script doesn't generate a message(i think it is because the dependency is test scope), so I minic the previous CHANGELOG pattern since it is test scope, and I think it should be different than the compile CHANGELOG message.